### PR TITLE
Review Required

### PR DESCRIPTION
**Pull request by cxl1521**
please check the added test method @cxl4252

### CODE REVIEW

File: TrangleTest.java [hunk 1]
```diff
@@ -16,5 +16,8 @@ public void setup() {
 		public void test() {
 			assertEquals(t.Tritype(0, 0, 0), 3);
 		}
-
+	
+		public void test2() {
+			assertEquals(t.Tritype(2, 3, 4), 4);
```
Comment (cxl4252) [inline TrangleTest.java:21]: Are you sure that the expected answer is correct

**Review by cxl4252 — COMMENTED** [review]
review test

File: TrangleTest.java (hunk 1 above)
Comment (cxl1521) [reply TrangleTest.java:21]: Yes. I am quite sure. Thanks.

Comment (cxl4252) [reply TrangleTest.java:21]: Ok. It can be committed.

Comment (cxl4252) [reply TrangleTest.java:21]: You may close the pull request.

Comment (cxl4252) [inline TrangleTest.java:21]: **cxl4252** 

> another problem is that the number of test methods is not enough

**Review by cxl4252 — APPROVED** [review]

**Comment by cxl1521** [comment]
complete a pull request with a review conversation

**Comment by cxl1521** [comment]
merge the 2nd update

**Pull request merged**